packages/processor/test/__snapshots__/composition.test.js.snap

```diff
@@ -71,10 +71,31 @@ Object {
 }
 `;
 
-exports[`/processor.js composition should compose from other files 1`] = `
+exports[`/processor.js composition should compose from other files in multiple declarations 1`] = `
+Object {
+  "packages/processor/test/specimens/composes2.css": Object {
+    "composes": Array [
+      "mc_foo",
+      "mc_folder2",
+      "mc_composes",
+    ],
+  },
+  "packages/processor/test/specimens/folder/folder2.css": Object {
+    "folder2": Array [
+      "mc_folder2",
+    ],
+    "foo": Array [
+      "mc_foo",
+    ],
+  },
+}
+`;
+
+exports[`/processor.js composition should compose from other files in single declaration 1`] = `
 Object {
   "packages/processor/test/specimens/composes.css": Object {
     "composes": Array [
+      "mc_foo",
       "mc_folder2",
       "mc_composes",
```

Comment:
Same question on this, I'd expect `folder2`, `foo`, `composes` but this has the first two swapped.

Reply:
I feel like it suppose to begin with `composes` so `composes, folder2, foo` would be an expected order. 🤔

Reply:
`composes` is the class itself, so it coming last is correct. I generally conceive of these things going in definition order, so `folder2` should be before `foo`.

Reply:
For me starting with `composes` class feels more natural. It resembles an order from a class rule in CSS:
```css
.composes {
    composes: folder2 from "./folder/folder2.css";
    composes: foo from "./folder/folder2.css";
}
```

Or maybe it's a bias learned from webpack's css-loader. 🤔